src/config.js

```diff
@@ -1,5 +1,7 @@
-// Load config
-var config = require('rc')("login-dc", {
+// Load config from config-file.
+const appName = 'dc-login-bot'
```

Comment:
so my old config file wont work anymore?
Also you might wanna add this file to the .gitgnore too, if you really want to rename it.

Reply:
we could probably rename the existing config file. But yeah, .gitignore makes sense.